public/js/test.js

```diff
@@ -50,3 +50,7 @@ function simulateBreach() {
 function doOauth() {
   window.open("/oauth/init");
 }
+
+$(document).foundation();
+
+document.querySelector("#subscribe-fxa-btn").addEventListener("click", doOauth);
```

Comment:
Sorry, I'm not 100% on how the `<script defer>` attribute works. Do we know that the DOM will have the `#subscribe-fxa-btn` rendered by the time the deferred `<script>` tag is eventually loaded? Or do we need to throw this in a jQuery `$(document).ready()` type thing?

Reply:
Nit: should this be jQuery-ified? A la `$("#subscribe-fxa-btn").click(doAuth);`, or something.

Reply:
In local testing, yes: with the `defer` attribute, the `#subscribe-fxa-btn` element is available by the time `test.js` accesses it. Without the `defer` attribute, it's broken.

I want to use `document.querySelector` in our own code so that Foundation is the only code that depends on jQuery. So if we drop Foundation we can also drop jQuery.